app/src/main/java/com/ndadheech/android/howdy/Friend.java

```diff
@@ -6,22 +6,22 @@
  * @author ndadheech
  */
 public class Friend {
-    private String friendName;
-    private String phoneNumber;
+    private String mFriendName;
```

Comment:
"m" indicates that the variable is a member variable. It truly is a current android convention: https://source.android.com/source/code-style.html#follow-field-naming-conventions. I'd prefer to use "m" prefixing since it is convention and that's how it was taught in the bootcamp.